src/axom/mir/examples/concentric_circles/mir_concentric_circles.cpp

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
```

Comment:
Please update the copyright years on all new files

Reply:
I think I got them all.